knack/index.css

```diff
@@ -0,0 +1,43 @@
+/* hide attachment type column (field contents are extracted via JS */
+div.view_2107 .field_2403 {
+  display: none;
+}
+div.view_2108 .field_2403 {
+  display: none;
+}
+
+/* hide attachment type columns (in Traffic Counts), field contents extracted via JS */
+div.view_2357 .field_3174 {
+  display: none;
+}
+div.view_2486 .field_3174 {
+  display: none;
+}
+
+/* hide attachment type columns (in Manage Requests under Traffic Counts), field contents extracted via JS */
+div.view_2491 .field_3174 {
+  display: none;
+}
+
+/* hide attachment type columns (in Request Status under Traffic Counts), field contents extracted via JS */
+div.view_2465 .field_3174 {
+  display: none;
+}
+
+/* move and style lat/lon fields and button to overlay React app map */
```

Comment:
I wonder if it would be nice, for the sake of documentation, to add more about where these classes are used.

Reply:
Agreed - it would be very hard to find these targets after moving on from this issue. I added the name of the page and scene id to the comment. Let me know if any other info would help, and I'll add.